scripts/symbolize.py

```diff
@@ -81,6 +82,15 @@ def __init__(self, out, dirs, strip_path):
         self._addr2line = None
         self.reset()
 
+    def my_Popen(self, cmd):
```

Comment:
Later when using Python > 3.3, then one could write a "check for all necessary tools" function using https://docs.python.org/3/library/shutil.html (shutil.which) instead.